docs/DataDictionary.md

```diff
@@ -0,0 +1,137 @@
+| EntityType      |     String      | Entity type used for filtering  | |
+| Version         |     String      | Version of LodeRunner being used   | | 
+| Id              |     String      | Unique Id generated at start-up to differentiate clients located in the same Region and Zone | |
+| Region          |     String      | The region in which the client is deployed | |
+| Zone            |     String      | The zone in which the client is deployed | |
+| Prometheus      |     Boolean     | Indicates whether or not this instance of LodeRunner is providing Prometheus metrics | |
+| StartupArgs     |     String      | String of arguments passed to LodeRunner at start-up | |
+| StartTime       |     DateTime    | The date and time this instance was started | |
+
+`Table 03: Load Client Properties`
+
+#### 2.2.3 ClientStatus Definition
+
+This object is primarily for conveying the curent status, time of that status, and the `LoadClient` settings to consuming apps.  It inherits from `BaseEntity` and contains a `LoadClient` member.
+
+##### ClientStatus
+
+| Property        |      Type       | Description                        | Notes      |
+| :-------------- | :-------------- | :--------------------------------- | :----------|
+| PartitionKey    |     String      | This value should be populated for `ClientStatus` objects and documents |  |
+| EntityType      |     String      | Entity type used for filtering  | [`ClientStatus`, `LoadTestConfig`, `TestRun`] |
+| LastUpdated     |     DateTime    | This shows the date and time the status was last updated | 
+| StateDuration   |     Int         | The number of seconds since the last change in state for the client | |
+| Status          |     string      | Current status of load client      | [`Starting`, `Ready`, `Testing`, `Terminating`] |
+| Message         |     string      | Additional information conveyed as part of the status update | |
+| LoadClient      |   `LoadClient`  | A nested object holding the information about the particular client in this status message | |
+
+`Table 04: ClientStatus Properties`
+
+### 2.3 Testing and Results Entities
+
+These are used for configuring a testing scenario.  `LoadTestConfig` will contain the settings that dictate what will be tested, with which files, and in what manner.  `TestRun` is used to schedule work with load clients and contains a `LoadTestConfg` and a list of LoadClients to use.
+
+#### 2.3.1 LoadTestConfig
+
+| Property        |    Type    | Description             | Required  | Notes      |
+| :-------------- | :--------- | :---------------------- | :-------- | :----------|
+| PartitionKey    |   String   | This value should be populated for `LoadTestConfig` objects and documents | Yes | |
+| EntityType      |   String   | Entity type used for filtering  | Yes | [`ClientStatus`, `LoadTestConfig`, `TestRun`] |
+| Files           |   String[]   | List of files to test | Yes | match `--files` CLI flag |
```

Comment:
Can we add a name property here so that the user of the UI can easily identify desired config

Reply:
@abassey2, good catch.  I added `Id` and `Name` to the base entity and then added those fields to each of the other tables.  So, we should now have those fields on each entity.